storage/innobase/log/log0online.cc

```diff
@@ -1317,13 +1317,11 @@ static void log_online_parse_complete_recs_past_previous_checkpoint() noexcept {
 page bitmap which is then written to disk.
 
 @return true if log tracking succeeded, false if bitmap write I/O error */
-bool log_online_follow_redo_log(void) {
+bool log_online_follow_redo_log_one_pass(void) {
```

Comment:
Also, can this one be made static?

Reply:
`srv_redo_log_follow_thread` is using `...one_pass` and I did not want change it. My original idea was to only change FLUSH command....

Reply:
Sure, no need to change that